docs/identity.md

```diff
@@ -0,0 +1,399 @@
+certificate has the Identity encoded in the Subject of the Certificate (with the
+Trust domain) and the certificate is signed by the Root CA.  When a pod communicates with
+another pod, `Pod A`'s Envoy instance will present its certificate
+to `Pod B`. `Pod B`'s Envoy instance will verify the
+Certificate was signed by a Certificate Authority it trusts and vise versa.
+This use of certificates is how mTLS is established and can be used for [Traffic Policies](https://release-v1-2.docs.openservicemesh.io/docs/getting_started/traffic_policies/#traffic-policies).
+
+> note: Although, OSM customers are primarily concerned about Service to Service
+> communication, OSM control plane components need to verify communication is trusted between them as
+> well.  Each webhook, and OMS control component will get a signed certificate as well.  
+
+Learn more about [Certificate issuance and usage](certificate_management.md).
+
+## Establishing a Pods Identity
+
+### Pod Admission
+When pod is created, [OSM's Mutating Webhook](https://github.com/openservicemesh/osm/blob/main/DESIGN.md#pod-lifecycle) injects Envoy as as side car with a
+bootstrap `Proxy Certificate`. This [Proxy
+Certificate](../DESIGN.md#b-proxy-tls-certificate) is used to gather information
+from the [Envoy xDS server in OSM control
+plane](https://github.com/openservicemesh/osm/blob/main/DESIGN.md#1-proxy-control-plane) and allows for dynamically updating which endpoints a Proxy can talk to, enables service certificate rotations and more. Learn more about the [design of
+OSM](../DESIGN.md) which goes into more details on how this works.
+
+#### Envoy Sidecar
+During the Mutating webhook step, the Pod Identity is established using the
+Pod's Service account and namespace.  The Envoy `Proxy Certificate` has the Pod
+Identity encoded in the Common Name (CN) field and DNS Subject Alternative Name (SAN) field of the certificate and is later
+used to confirm Pod Identity and issue certificates for Pod to Pod
+communication. 
+
+The Envoy `Proxy Certificate` CN and SAN DNS name has the following format:
+
+```
+<proxy-UUID>.<kind>.<proxy-identity>.<trust-domain>
+```
+
+- `proxy-uuid` = random unique GUID
+- `kind` = `sidecar` 
+- `proxy-identity` = For Kubernetes this identity is from the pod spec
+  in the form of `<service-account>.<namespace>`
+- `trust-domain` = default is `local.cluster`
+
+An example of this cert can be seen by forwarding the Envoy Admin UI:
+
+```bash
+kubectl port-forward bookwarehouse-78757788bb-zs4hk 15000
+Forwarding from 127.0.0.1:15000 -> 15000
+Forwarding from [::1]:15000 -> 15000
+
+# from another terminal
+curl http://localhost:15000/config_dump | jq -r '.configs[] | select(."@type"=="type.googleapis.com/envoy.admin.v3.SecretsConfigDump") | .dynamic_active_secrets[] | select(.name=="tls_sds").secret.tls_certificate.certificate_chain.inline_bytes' | base64 -d | openssl x509 -noout -text
```

Comment:
`select` seems like a powershell command. Could you simply use `jq` instead?

Reply:
select is jq function: https://stedolan.github.io/jq/manual/#select(boolean_expression)

Reply:
Perfect!